### ISSUE TITLE
[Enhancement] Validation popup on Sonoma Public Beta, add macOS 14 to tested version range

### ISSUE THREAD

**Issue by taninke** [LiveEnhancementSuite/LESforMacOS#27]
Easy enough to reproduce!

Simply install the macOS beta software (I know, I know), and this will popup.

Beyond this, functionality works fine, just wanted to make people aware of this popup.

<img width="257" alt="Screenshot 2023-07-13 at 8 12 34 PM" src="https://github.com/LiveEnhancementSuite/LESforMacOS/assets/28825177/57d7726f-92bc-4cf1-85bb-da6f0d4beb79">

 

**Desktop (please complete the following information):**
 - OS: macOS Sonoma 14.0 Beta (23A5286i)

**Comment by kwvg** [comment]
Not a bug, that's an intended feature. 

Until I can get enough testers to confirm it works on Sonoma _after_ it's released as stable (or I upgrade to Sonoma and test it), that warning is there to stay. 

_But_, that warning is meant for people who aren't likely to install a beta... for those who are technical and are aware of the risks involved, you can edit your settings and set `checksanity` to `0` to bypass the prompt.

**Comment by taninke** [comment]
> Not a bug, that's an intended feature.
> 
> Until I can get enough testers to confirm it works on Sonoma _after_ it's released as stable (or I upgrade to Sonoma and test it), that warning is there to stay.
> 
> _But_, that warning is meant for people who aren't likely to install a beta... for those who are technical and are aware of the risks involved, you can edit your settings and set `checksanity` to `0` to bypass the prompt.

Makes sense, thanks for the response! If I get any legitimate bugs I’ll make sure to report them!
